spec/jobs/performance_platform_transactions_queue_job_spec.rb

```diff
@@ -14,15 +24,18 @@
     expect(PerformancePlatformTransactionsQueueJob.new.queue_name).to eq('performance_platform')
   end
 
-  it 'executes perform' do
+  it 'enqueues a job to send all published job from yesterday' do
     stub_const('PP_TRANSACTIONS_BY_CHANNEL_TOKEN', 'not-nil')
 
     pp = double(:performance_platform)
 
-    published = create_list(:vacancy, 3, :published, publish_on: date)
+    job_1_published_yesterday = build(:vacancy, :past_publish, publish_on: date_to_upload).save(validate: false)
+    job_2_published_yesterday = build(:vacancy, :past_publish, publish_on: date_to_upload).save(validate: false)
+    jobs_published_yesterday = [job_1_published_yesterday, job_2_published_yesterday]
```

Comment:
Probably best to add some jobs published today and further in the past to check they don't get submitted